libraries/AP_NavEKF3/AP_NavEKF3_Control.cpp

```diff
@@ -361,7 +361,7 @@ void NavEKF3_core::setAidingMode()
             } else if (readyToUseExtNav()) {
                 // we are commencing aiding using external nav
                 posResetSource = EXTNAV;
-                velResetSource = DEFAULT;
+                velResetSource = EXTNAV;
```

Comment:
As discussed on the dev call this is probably incorrect.  We probably need an additional check that we have received velocity data.  If "yes" then we set the velResetSource to EXTNAV, if not we leave it as DEFAULT.

Reply:
I've added a commit, "AP_NavEKF3: velocity reset when using extNav uses default method " that resolves this and tested in SITL to ensure that it does reset to the external nav velocities if present.